packages/database/prisma/migrations/20230921110737_initial/migration.sql

```diff
@@ -0,0 +1,271 @@
+-- CreateTable
+CREATE TABLE `Study` (
+    `id` INTEGER NOT NULL,
+    `name` VARCHAR(191) NOT NULL,
+    `status` VARCHAR(191) NOT NULL,
+    `recordStatus` VARCHAR(191) NOT NULL,
+    `route` VARCHAR(191) NOT NULL,
+    `irasId` VARCHAR(191) NULL,
+    `protocolReferenceNumber` VARCHAR(191) NULL,
+    `sampleSize` INTEGER NULL,
+    `chiefInvestigatorFirstName` VARCHAR(191) NULL,
+    `chiefInvestigatorLastName` VARCHAR(191) NULL,
+    `managingSpeciality` VARCHAR(191) NOT NULL,
+    `plannedOpeningDate` DATETIME(3) NULL,
+    `plannedClosureDate` DATETIME(3) NULL,
+    `actualOpeningDate` DATETIME(3) NULL,
+    `actualClosureDate` DATETIME(3) NULL,
+    `createdAt` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updatedAt` DATETIME(3) NOT NULL,
+    `isDeleted` BOOLEAN NULL DEFAULT false,
+
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `Organisation` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(191) NOT NULL,
+    `rtsIdentifier` VARCHAR(191) NOT NULL,
```

Comment:
Whats the rationale for 191?

Reply:
That's the default Prisma gives for mysql text fields, I believe for some legacy reasons around indexing, we can increase if necessary.